src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts

```diff
@@ -329,24 +330,27 @@ class ZWaveJSNodeConfig extends SubscribeMixin(LitElement) {
   }
 
   private async _fetchData() {
-    if (!this.configEntryId) {
+    if (!this.configEntryId || !this._deviceRegistryEntries) {
       return;
     }
 
+    const getDevice = memoizeOne((entries, deviceId) =>
```

Comment:
This won't work because you re-define this function every time so it won't be able to memoize. Move this above the class definition. Don't forget the types :)

Reply:
😅